test/local/senders/email.js

```diff
@@ -832,6 +832,12 @@ describe(
             assert.equal(mailerSend1.args[0].to, message.email, 'logs sender to email address')
           })
       })
+    })
+
+    describe('mock failing sendMail method:', () => {
+      beforeEach(() => {
+        sinon.stub(mailer.mailer, 'sendMail', (config, cb) => cb(new Error('Fail')))
+      })
```

Comment:
This is the fix for the weird test that I think should have been failing before. The test assumes that `sendMail` is going to fail, but the setup above mocks it to succeed. I can only assume that the `assert.notOk` was not being executed somehow, or maybe it was being executed but it's error wasn't being propagated back?

Whatever, the correct thing to do seemed to be to put it in its own setup with a `sendMail` that's mocked to fail?